src/clientdlg.cpp

```diff
@@ -375,6 +375,17 @@ CClientDlg::CClientDlg ( CClient*         pNCliP,
     pMenu->addMenu ( pEditMenu );
     pMenu->addMenu ( new CHelpMenu ( true, this ) );
 
+#if defined( Q_OS_IOS ) or defined( Q_OS_ANDROID ) or defined( ANDROID )
```

Comment:
I'd suggest this whole change - switching input device - be split out from the main pull request.  It will make more sense as a separate change to have both iOS and android support then.  Here the android bit looks lost.